tsdb/query_executor.go

```diff
@@ -242,11 +242,22 @@ func (q *QueryExecutor) PlanSelect(stmt *influxql.SelectStatement, chunkSize int
 	// Replace instances of "now()" with the current time, and check the resultant times.
 	stmt.Condition = influxql.Reduce(stmt.Condition, &influxql.NowValuer{Now: now})
 	tmin, tmax := influxql.TimeRange(stmt.Condition)
+	explicitTimeRange := true
 	if tmax.IsZero() {
 		tmax = now
+		explicitTimeRange = false
 	}
 	if tmin.IsZero() {
 		tmin = time.Unix(0, 0)
+		explicitTimeRange = false
+	}
+
+	// for queries with small group by interval set time range if it wasn't explicit. #4481
```

Comment:
Can you pull the bug number from here? We don't tend to do that in our code, I don't think you'll find it anywhere else.